config/environments/development.rb

```diff
@@ -27,11 +42,13 @@
   # number of complex assets.
   config.assets.debug = true
 
-  # Adds additional error checking when serving assets at runtime.
-  # Checks for improperly declared sprockets dependencies.
-  # Raises helpful error messages.
-  config.assets.raise_runtime_errors = false
+  # Suppress logger output for asset requests.
+  config.assets.quiet = true
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  # Use an evented file watcher to asynchronously detect changes in source code,
+  # routes, locales, etc. This feature depends on the listen gem.
+  # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
```

Comment:
Seems strange to have this added, but commented out.

Reply:
It’s the built-in `app:update` command, I think. I agree that it’s suboptimal for that thing to add commented out lines. I guess the rails folks wanted to make users aware of the options and expected them to manually make updates. I’m lamenting not having open sourced our rails upgrade scripts before we stopped focusing on rails stuff, since they were significantly better than the framework-provided commands.

Reply:
@stevecrozz strange? Ok, it will be removed in next rails upgrade. But, If you like, I will add listen gem and everything around.

@burnettk Yep, you are completely right. This is `app:update` rake task.

Reply:
@stevecrozz I add listen and uncomment this code.